modules/jaggeryapps/ml/site/project/compare.jag

```diff
@@ -183,10 +190,74 @@
         '<p>' + message + '</p>'
         '</div>';
         return builtErrors;
-    }     
-
-    function loadModels() {
-        
+    }
+    // compare function for classification models
+    function sortByAccuracy(a, b) {
+        var sortStatus = 0;
+     
+        if (a.modelSummary.modelAccuracy > b.modelSummary.modelAccuracy) {
+            sortStatus = -1;
+        } else if (a.modelSummary.modelAccuracy < b.modelSummary.modelAccuracy) {
+            sortStatus = 1;
+        }
+        return sortStatus;
+    }
+    // compare function for numerical prediction models
+    function sortByError(a, b) {
+        var sortStatus = 0;
+     
+        if (a.modelSummary.meanSquaredError < b.modelSummary.meanSquaredError) {
+            sortStatus = -1;
+        } else if (a.modelSummary.meanSquaredError > b.modelSummary.meanSquaredError) {
+            sortStatus = 1;
+        }
+        return sortStatus;
+    }
```

Comment:
Seems we have to keep the sorting logic at the front end for now, till we figure out a way to represent these model accuracy parameters in a generic manner.